### PR TITLE
refac: updated events.drawio.png

### PR DESCRIPTION
**Pull request by energinet-ajw**
Updated events.drawio.png.

## Description

events drawio

## References

* #887

### CODE REVIEW

File: docs/images/events.drawio.png
Comment (BjarkeMeier) [inline docs/images/events.drawio.png:1]: My thoughts:
- Transparent doesn't work well with dark theme. Please keep the white background
- Do you think it's better without the legend?
- It displays nicer if you keep the margins (especially in dark theme)
- Change "calculation completed" to "calculation result completed"
- How about also visually indicating that there are many calculate result completed events per action?

Comment (energinet-ajw) [reply docs/images/events.drawio.png:1]: Updated: white background, legend added, visually indicating many events, "calculation result completed" used and added margins.

Comment (BjarkeMeier) [inline docs/images/events.drawio.png:1]: Are we missing a batch-created event?
Can we move "events" in the sentences of the 5 publish-calculation-result completed boxes?

Comment (energinet-ajw) [reply docs/images/events.drawio.png:1]: Done.

**Review by lasrinnil — COMMENTED** [review]
Perhaps we should distinguish between internal and external events.

**Review by JesperGomes — APPROVED** [review]

**Pull request merged**